llama_stack/apis/inference/client.py

```diff
@@ -100,25 +101,31 @@ async def chat_completion(
                             print(f"Error with parsing or validation: {e}")
 
 
-async def run_main(host: str, port: int, stream: bool):
+async def run_main(host: str, port: int, stream: bool, model: str):
     client = InferenceClient(f"http://{host}:{port}")
 
+    if not model:
+        model = "Llama3.1-8B-Instruct"
+
     message = UserMessage(
         content="hello world, write me a 2 sentence poem about the moon"
     )
     cprint(f"User>{message.content}", "green")
     iterator = client.chat_completion(
-        model="Llama3.1-8B-Instruct",
+        model=model,
         messages=[message],
         stream=stream,
     )
     async for log in EventLogger().log(iterator):
         log.print()
 
 
-async def run_mm_main(host: str, port: int, stream: bool, path: str):
+async def run_mm_main(host: str, port: int, stream: bool, path: str, model: str):
```

Comment:
nit: (also existing issue) -- the typehint for both `path` and `model` should be `Optional[str]`

Reply:
yeah, I noticed a warning because of that. I can go ahead and fix it here.